Packs/CommonPlaybooks/Playbooks/playbook-Cloud_User_Investigation_-_Generic.yml

```diff
@@ -529,6 +529,67 @@ inputs:
   required: false
   description: The cloud service provider involved.
   playbookInputQuery:
+inputSections:
+- inputs:
+  - AzureSearchTime
+  - failedLogonThreshold
```

Comment:
-  failedLogonThreshold
  - MfaAttemptThreshold
Aren't they supposed to be under the incident data section? 
Also, incident data should be the first one section

Reply:
failedLogonThreshold and MfaAttemptThreshold are part of the verdict threshold under the investigation. so I think they should stay in the investigation section.